### PR TITLE
efibootmgr: new package

### PR DESCRIPTION
**Pull request by oskarirauta**
Maintainer: me / @oskarirauta 
Compile tested: x86-64, recent git
Run tested: x86-64, recent git

Description:
Linux user-space application to modify the Intel Extensible Firmware Interface (EFI) Boot 
Manager. This application can create and destroy boot entries, change the boot order, change the next running 
boot option, and more.

Depends on my another PR #20276 (efivar library)

Signed-off-by: Oskari Rauta <oskari.rauta@gmail.com>

### CODE REVIEW

**Comment by oskarirauta** [comment]
Build ofcourse won't work until #20276 is merged, but could someone please help a bit here, what are these ci errors that I am getting with nearly every PR that I've recently made?

**Comment by oskarirauta** [comment]
@1715173329 

This looks good to go too, maybe check this too after efivar library..? :)
This is the utility used by most distributions to modify boot order/options with efi.

File: utils/efibootmgr/Makefile
Comment (1715173329) [inline utils/efibootmgr/Makefile:18]:
```suggestion
PKG_LICENSE:=GPL-2.0-only
PKG_LICENSE_FILES:=COPYING
PKG_MAINTAINER:=OSkari Rauta <oskari.rauta@gmail.com>

PKG_BUILD_PARALLEL:=1
PKG_INSTALL:=1
```

File: utils/efibootmgr/Makefile
Comment (1715173329) [inline utils/efibootmgr/Makefile:41]:
```suggestion
MAKE_VARS += \
	EFIDIR="/boot/EFI" \
	PCDIR="$(STAGING_DIR)/usr/lib/pkgconfig" \
	EXTRAINCDIR="-I$(STAGING_DIR)/usr/include -I$(STAGING_DIR)/usr/include/efivar" \
	EXTRALIBDIR="-L$(STAGING_DIR)/usr/lib"
```

File: utils/efibootmgr/Makefile
Comment (1715173329) [inline utils/efibootmgr/Makefile:25]: I think it would be too long for menuconfig, can you try to shorten it?

File: utils/efibootmgr/Makefile
Comment (1715173329) [inline utils/efibootmgr/Makefile:31]: please wrap a new line after every 70 characters.

**Review by 1715173329 — CHANGES_REQUESTED** [review]

File: utils/efibootmgr/Makefile
Comment (oskarirauta) [reply utils/efibootmgr/Makefile:25]: Sure thing

File: utils/efibootmgr/Makefile
Comment (oskarirauta) [reply utils/efibootmgr/Makefile:31]: Consider it as done :)

**Comment by oskarirauta** [comment]
@1715173329 

This also says that change request is pending- but as far as I know, all requsted changes should now be resolved. Could you verify if I have missed one?

**Comment by oskarirauta** [comment]
Once efivar is merged, this should be possible.

**Review by 1715173329 — APPROVED** [review]

**Comment by oskarirauta** [comment]
refind_screenshot

Screenshot of rEFInd, installed as a (default) boot option with efibootmgr.

**Pull request merged**
